examples/petsc/bp3.h

```diff
@@ -81,12 +81,27 @@ CEED_QFUNCTION(Diff)(void *ctx, const CeedInt Q,
   // Quadrature Point Loop
   CeedPragmaOMP(simd)
   for (CeedInt i=0; i<Q; i++) {
-    const CeedScalar ug0 = ug[i+Q*0];
-    const CeedScalar ug1 = ug[i+Q*1];
-    const CeedScalar ug2 = ug[i+Q*2];
-    vg[i+Q*0] = qd[i+Q*0]*ug0 + qd[i+Q*1]*ug1 + qd[i+Q*2]*ug2;
-    vg[i+Q*1] = qd[i+Q*1]*ug0 + qd[i+Q*3]*ug1 + qd[i+Q*4]*ug2;
-    vg[i+Q*2] = qd[i+Q*2]*ug0 + qd[i+Q*4]*ug1 + qd[i+Q*5]*ug2;
-  }
+    // Read spatial derivatives of u
+    const CeedScalar uJ[3]        =  {ug[i+Q*0],
+                                      ug[i+Q*1],
+                                      ug[i+Q*2]
+                                     };
+    // Read qdata
+    const CeedScalar dXdxdXdxT[6] =  {qd[i+0*Q],
+                                      qd[i+1*Q],
+                                      qd[i+2*Q],
+                                      qd[i+3*Q],
+                                      qd[i+4*Q],
+                                      qd[i+5*Q]
+                                     };
+
+    const CeedInt idx[3][3] = {{0, 1, 2}, {1, 3, 4}, {2, 4, 5}}; // symmetric matrix indices
+
+    for (int j=0; j<3; j++) {
+      vg[i+(0+j*3)*Q]  = 0;
```

Comment:
Change to `[i+Q*j]`?

Reply:
Sorry that was just a bad commit that had slipped through